pyaedt/modules/Boundary.py

```diff
@@ -643,146 +847,140 @@ def override_component(
-            Surface material to apply to parts. Default is ``"Steel-oxidised-surface"``.
+        reference_designator : str
+            Reference designator of the instance to override.
+        filter_component : bool, optional
+            Whether to filter out the component. Default is ``False``.
+        power : str, optional
+            Override component power. Default is ``None``, in which case the power is not overridden.
+        r_jb : str, optional
+            Override component r_jb value. Default is ``None``, in which case the resistance is not overridden.
+        r_jc : str, optional
+            Override component r_jc value. Default is ``None``, in which case the resistance is not overridden.
+        height : str, optional
+            Override component height value. Default is ``None``, in which case the height is not overridden.
 
         Returns
         -------
         bool
             ``True`` if successful. ``False`` otherwise.
         """
-        self.props["NativeComponentDefinitionProvider"]["PartsChoice"] = 1
-        self.props["NativeComponentDefinitionProvider"]["ModelDeviceAsRect"] = simplify_parts
-        self.props["NativeComponentDefinitionProvider"]["DeviceSurfaceMaterial"] = surface_material
-        return True
+        return self._override_common(
+            "instanceOverridesMap",
+            reference_designator=reference_designator,
+            filter_component=filter_component,
+            power=power,
+            r_jb=r_jb,
+            r_jc=r_jc,
+            height=height,
+        )
 
-    @pyaedt_function_handler()
-    @disable_auto_update
-    def set_package_parts(
-        self,
-        solderballs=None,
-        connector=None,
-        solderbumps_modeling="Boxes",
-        bondwire_material="Au-Typical",
-        bondwire_diameter="0.05mm",
-    ):
-        """Set how to include PCB device parts.
 
-        Parameters
-        ----------
-        solderballs : str, optional
-            Specifies whether the solderballs located below the stackup are modeled,
-            and if so whether they are modeled as ``"Boxes"``, ``"Cylinders"`` or ``"Lumped"``.
-        connector : str, optional
-            Specifies whether the connectors located above the stackup are modeled,
-            and if so whether they are modeled as ``"Solderbump"`` or ``"Bondwire"``.
-            Default is ``None`` in which case they are not modeled.
-        solderbumps_modeling : str, optional
-            Specifies how to model solderbumps if ``connector`` is set to ``"Solderbump"``.
-            Accepted options are: ``"Boxes"``, ``"Cylinders"`` and ``"Lumped"``.
-            Default is ``"Boxes"``.
-        bondwire_material : str, optional
-            Specifies bondwires material if ``connector`` is set to ``"Bondwire"``.
-            Default is ``"Au-Typical"``.
+class NativeComponentPCB(NativeComponentObject, object):
+    """Manages Native Component PCB data and execution.
```

Comment:
```suggestion
    """Manages native component PCB data and execution.
```
I don't believe "Native Component" is a proper noun and so should be in lowercase.